lib/vscode/package.json

```diff
@@ -116,7 +116,6 @@
     "css-loader": "^3.2.0",
     "debounce": "^1.0.0",
     "deemon": "^1.4.0",
-    "electron": "9.3.3",
```

Comment:
this was intentional, right?

Reply:
@jsjoeio see the per commit breakdown!

Reply:
Ah sorry! Should have looked there first. Got it 👍🏼